docs/src/main/sphinx/release/release-460.md

```diff
@@ -0,0 +1,52 @@
+# Release 460 (2 Oct 2024)
+
+## Atop connector
+
+* {{breaking}} Remove the Atop connector. ({issue}`23550`)
+
+## ClickHouse connector
+
+* Improve performance of listing columns. ({issue}`23429`)
```

Comment:
In what context? Which queries does it affect? Queries on `information_schema.columns`?

Reply:
Seems like all queries that need column info from looking at the PR so the generic text is correct.